docs/02.API-REFERENCE.md

```diff
@@ -35,7 +35,8 @@ Possible compile time enabled feature types:
  - JERRY_FEATURE_REGEXP_DUMP - regexp byte-code dumps
  - JERRY_FEATURE_SNAPSHOT_SAVE - saving snapshot files
  - JERRY_FEATURE_SNAPSHOT_EXEC - executing snapshot files
- - JERRY_FEATURE_DEBUGGER - debugging
+ - JERRY_FEATURE_DEBUGGER - debugger server is available
```

Comment:
Why this change? It is both unrelated and makes the typedef out of sync from the doc.